r-nsubuga/Sacco-Frontend
# Login

What does this PR do?
A user can login into the system

Description of Task to be completed?
Enter the username and password to access the system

How should this be manually tested?
Enter incorrect information to check validity and authentication

Screenshots (if appropriate)
![Screenshot from 2023-07-17 15-35-51](https://github.com/r-nsubuga/Sacco-Frontend/assets/96862761/2dd64692-0628-406b-ad4e-b88166641282)
